public/help/help.md

```diff
@@ -1 +1,27 @@
-#Instructions for this section
\ No newline at end of file
+#### Sequence Tube Map Usage Guide
+The Sequence Tube Map is used to generate visualizations of genomic sequence graphs. This guide will demonstrate how to select custom data to display visualizations. [Link to usage guide on Github](https://github.com/vgteam/sequenceTubeMap/blob/readme_updates/public/help/help.md)
```

Comment:
```suggestion
The Sequence Tube Map is used to generate visualizations of genomic sequence graphs. This guide will demonstrate how to select custom data to visualize. [You can also read this guide on Github.](https://github.com/vgteam/sequenceTubeMap/blob/master/public/help/help.md)
```

We probably want to link to where this is going to appear long-term, once we merge the PR, instead of to the branch.